tutorials/system-building-protein-ligand.ipynb

```diff
@@ -37,13 +37,16 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
+      "Loaded Config for HTMD @ Acellera Office\n",
```

Comment:
We have to be careful to not commit office specific configurations (no action needed, I'm fixing it myself)

Reply:
@j3mdamas sorry my fault